vendor/osxlockdown/commands.yaml

```diff
@@ -0,0 +1,276 @@
+  enabled: true
+
+
+- title: "Disable IPv6"
+  check_command: |
+    networksetup -listallnetworkservices | while read i; do SUPPORT=$(networksetup -getinfo "$i" | grep "IPv6: Automatic") && if [ -n "$SUPPORT" ]; then exit 1; fi; done; exit 0
+  fix_command: |
+    networksetup -listallnetworkservices | while read i; do SUPPORT=$(networksetup -getinfo "$i" | grep "IPv6: Automatic") && if [ -n "$SUPPORT" ]; then networksetup -setv6off "$i"; fi; done;
+  enabled: true
+
+
+- title: "Disable Previews"
+  check_command: |
+    defaults read /Library/Preferences/com.apple.finder.plist | grep ShowIconThumbnails | grep 0
+  fix_command: |
+    /usr/libexec/PlistBuddy -c "Add StandardViewOptions:ColumnViewOptions:ShowIconThumbnails bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewSettings:ListViewSettings:showIconPreview bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewSettings:IconViewSettings:showIconPreview bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewSettings:ExtendedListViewSettings:showIconPreview bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewOptions:ColumnViewOptions:ShowPreview bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewSettings:ListViewSettings:showPreview bool NO" "/Library/Preferences/com.apple.finder.plist" && /usr/libexec/PlistBuddy -c "Add StandardViewSettings:IconViewSettings:showPreview bool NO" "/Library/Preferences/com.apple.finder.plist" &&  /usr/libexec/PlistBuddy -c "Add StandardViewSettings:ExtendedListViewSettings:showPreview bool NO" "/Library/Preferences/com.apple.finder.plist"
+  comment: "TODO: I'm only checking one item, check all"
+  enabled: true
+
+
+- title: "Secure Safari by crippling it"
+  check_command: |
+    defaults read com.apple.Safari WebKitOmitPDFSupport | grep 1
+  fix_command: |
+    defaults write com.apple.Safari WebKitOmitPDFSupport -bool YES && defaults write com.apple.Safari WebKitJavaScriptEnabled -bool FALSE && defaults write com.apple.Safari com.apple.Safari.ContentPageGroupIdentifier.WebKit2JavaScriptEnabled -bool FALSE
+  comment: "I 'secure' safari by removing javascript and PDF support. Advanced users won't use Safari anyway and novices will be persuaded to use Chrome or Firefox.  TODO: This needs to be done per user."
+  enabled: true
+
+
+- title: "Disable automatic loading of remote content by Mail.app"
+  check_command: |
+    defaults read com.apple.mail-shared DisableURLLoading | grep 1
+  fix_command: |
+    defaults write com.apple.mail-shared DisableURLLoading -bool true
+  comment: "I improve security of OSX's Mail app by disabling automatic loading of remote content in opened emails. TODO: This needs to be done per user."
+  enabled: true
+
+
+- title: "Disable Captive Portal"
```

Comment:
@kamilhism looks like it's a good idea to enable this check https://github.com/drduh/OS-X-Security-and-Privacy-Guide#captive-portal. What do you think?